internal/pkg/cli/dialog/use_template.go

```diff
@@ -128,7 +128,7 @@ func (d *useTmplDialog) askInput(inputDef input.Input) error {
 
 	// Ask for input
 	switch inputDef.Kind {
-	case input.KindInput, input.KindPassword, input.KindTextarea:
+	case input.KindInput, input.KindHidden, input.KindTextarea:
```

Comment:
Password nie je najlepsi nazov, ... moze to byt napr. private SSH key, tak som to premenoval `passoword -> hidden`.